src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -30,6 +32,9 @@ namespace Microsoft.PowerShell.EditorServices.Commands
     [Cmdlet(VerbsLifecycle.Start, "EditorServices", DefaultParameterSetName = "NamedPipe")]
     public sealed class StartEditorServicesCommand : PSCmdlet
     {
+        // TODO: Remove this when we drop support for PS6
+        private static bool _isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
```suggestion
        private static bool s_isWindows = 
#if CoreCLR
        RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
#else
        true;
#endif
```

Reply:
added and also the other one for the using.